src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html

```diff
@@ -41,6 +41,37 @@
     </div>
     <div class="flex-horizontal flex-noshrink">
         <div class="flex-separator"></div>
+    <div class="btn-group dropdown-right">
+        <button class="btn btn-default btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"
+                title="Choose visible columns">
+            <i class="icon-table"></i><span>Display</span>
+            <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu colselect-dropdown slidein-style" role="menu" data-bind="dropdownPanel: true">
+            <div class="columns-list-container flex-vertical">
+                <div class="flex-header">
+                    <div class="checkbox">
+                        <input type="checkbox" id="select_all" class="styled" data-bind="checked: isSelectAllChecked, event: { change: toggleSelectAll }">
+                        <label for="select_all">
+                            Select All
+                        </label>
+                    </div>
+                </div>
+                <div class="flex-grow">
+                    <ul class="column-list scroll" data-bind="foreach: { data: filteredTypeData, as: 'item' }">
```

Comment:
we usually don't alias items in foreach loops - please use foreach: filteredTypeData

Reply:
done